pexpect/pty_spawn.py

```diff
@@ -155,6 +155,11 @@ def __init__(self, command, args=[], timeout=30, maxread=2000,
         then exitstatus will store the exit return code and signalstatus will
         be None. If the child was terminated abnormally with a signal then
         signalstatus will store the signal value and exitstatus will be None.
+        
+            child = pexpect.spawn('some_command')
```

Comment:
I think this looks helpful, but can you stick a `::` above it, with a blank line either side? That will make it show up as a code block [in the docs](http://pexpect.readthedocs.org/en/stable/api/pexpect.html#spawn-class).